src/components/TodoItem.tsx

```diff
@@ -0,0 +1,124 @@
+import React, {
+  useEffect,
+  useRef,
+  useState,
+  useContext,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodosContext } from './TodosContext';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { todos, setTodos, setToggleAllStatus } = useContext(TodosContext);
+  const [editStatus, setEditStatus] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isEditStatus, setIsEditStatus] = useState(false);
```

Reply:
<img width="639" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/70220667/e370cbc8-f693-481d-bdab-1d3dacfbe498">
still not fixed